metricbeat/module/vsphere/datastore/datastore_test.go

```diff
@@ -58,7 +58,9 @@ func TestFetchEventContents(t *testing.T) {
 		if err != nil {
 			t.Error(err)
 		} else {
-			isNonNegativeInt64(t, field, value)
+			if field != "capacity.used.pct" {
```

Comment:
You can remove `capacity.used.pct` from the list above.

Reply:
ahh didnt see the list :) will do!